lib/wordsmith/project.rb

```diff
@@ -11,20 +11,13 @@ def self.find(slug)
     project or fail %Q(Project not found with slug: "#{slug}")
   end
 
-  def schema
-    @_schema ||=
-      begin
-        body = Wordsmith.client.get "projects/#{slug}"
-        body[:schema]
-      end
-  end
-
   private
 
-  def initialize(name: nil, slug: nil, templates: nil)
-    raise "Missing required kword arguments" unless [name, slug, templates].all?
+  def initialize(name: nil, slug: nil, schema: nil, templates: nil, **attributes)
+    raise "Missing required keyword arguments" unless [name, slug, templates].all?
```

Comment:
@jongaegler I definitely debated about whether to add it.  The problem with adding schema is it would cause the sdk to break for the current version (v1.0) of the API.  Maybe that's not a concern if the sdk update is released after v1.1.  I'm not actually sure why we need to raise an error here anyway.